src/components/date-picker/date.types.ts

```diff
@@ -7,4 +7,13 @@ export type DateType =
     | 'quarter'
     | 'year';
 
-export type Languages = 'da' | 'en' | 'fi' | 'nb' | 'no' | 'nl' | 'sv';
+export type Languages =
+    | 'da'
+    | 'de'
+    | 'en'
+    | 'fi'
+    | 'fr'
+    | 'nb'
+    | 'no'
+    | 'nl'
+    | 'sv';
```

Comment:
This is out of scope for this PR, but I think we should deprecate this type and change the prop on the component to a string instead. The user could then use any locale they want without us having to update and maintain this list every time we need a new locale. It will be hard to do as long as we depend on Moment and all the locale imports it requires, but if we switch from Moment to `Intl.DateTimeFormatter` it would probably not be too hard.

Reply:
Yeah, that's good input. I don't like that we do this either, but with Moment it felt like a necessity to not just import all available localisations.

In case `Intl.DateTimeFormat` doesn't solve the problem completely (it doesn't support week and quarter formats, for example), we could perhaps solve it by lazy-loading the requested localisation, instead of bundling them all…? 🤔

Reply:
> No approval though? grimacingsmile

Oh sorry, I got distracted by other stuff :smile: